packages/automator/index.tsx

```diff
@@ -273,27 +320,31 @@ const batchProcess = async (
   // Determine the output file path
   const folders = args["--folders"] || false;
   const browserFolder = args["--render"];
-  const outFile = args["--outFile"];
+  const outFile = args["--outFile"] || `${args.OUTFOLDER}/output.svg`;
   const times = args["--repeat"] || 1;
   const prefix = args["--src-prefix"];
 
+  const staged = args["--staged"] || false;
+
   if (args.batch) {
     for (let i = 0; i < times; i++) {
-      await batchProcess(args.LIB, folders, args.OUTFOLDER, prefix);
+      await batchProcess(args.LIB, folders, args.OUTFOLDER, prefix, staged);
     }
     if (browserFolder) {
       renderArtifacts(args.OUTFOLDER, browserFolder);
     }
   } else if (args.render) {
     renderArtifacts(args.ARTIFACTSFOLDER, args.OUTFOLDER);
   } else {
+    // this assumes that yarn start draw was called.
```

Comment:
Now that I think about this, I think it's probably better if we do `else if (args.draw)` and return an error on `else`.